tests/Aspire.Templates.Tests/AppHostTemplateTests.cs

```diff
@@ -1,6 +1,9 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+// TODO: Fix this.
+#pragma warning disable xUnit1051 // Calls to methods which accept CancellationToken should use TestContext.Current.CancellationToken to allow test cancellation to be more responsive.
```

Comment:
Disable this in the `Aspire.Templates.Tests.csproj` instead of adding to all the test files.

Reply:
I opened https://github.com/dotnet/aspire/issues/8488 and pushed the change. Please feel free to discuss if you don't agree with the change.